datastructures/clockchain.py

```diff
@@ -1,19 +1,24 @@
 from utils.helpers import hasher, measure_tick_continuity
 from utils.common import logger, credentials, config
 from utils.pki import pubkey_to_addr
+from utils.validation import validate_clockchain
 from queue import Queue, PriorityQueue
 import copy
 import time
+import requests
 
 
 class Clockchain(object):
-    def __init__(self):
+    def __init__(self, networker):
```

Comment:
I made Clockchain and Networker as "pure" instances that are not used by eachother. This avoids spaghetti code later on

Please try to see if there's any way for clockchain to not use Networker

Reply:
I agree that would be cleaner, but the clockchain needs access to the urls of the peers from which to copy the chain. What do you suggest?

Reply:
discussed we can just move it to timeminer etc